<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectImportMethod</class>
 <widget class="QDialog" name="SelectImportMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>131</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>181</width>
    <height>131</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>181</width>
    <height>131</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonImportDocs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Archivo(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonImportDB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Base de datos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonImportTotalNatDoc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CC Nacional</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
